src/libs/WorkflowUtils.ts

```diff
@@ -191,22 +196,48 @@ function convertApprovalWorkflowToPolicyEmployees({approvalWorkflow, membersToRe
 
     approvalWorkflow.approvers.forEach((approver, index) => {
         const nextApprover = approvalWorkflow.approvers.at(index + 1);
+        const forwardsTo = type === CONST.APPROVAL_WORKFLOW.TYPE.REMOVE ? '' : nextApprover?.email ?? '';
+
+        if (previousEmployeeList[approver.email]?.forwardsTo === forwardsTo) {
+            return;
+        }
+
         updatedEmployeeList[approver.email] = {
             email: approver.email,
-            forwardsTo: type === CONST.APPROVAL_WORKFLOW.TYPE.REMOVE ? '' : nextApprover?.email ?? '',
+            forwardsTo,
         };
     });
 
     approvalWorkflow.members.forEach(({email}) => {
+        const submitsTo = type === CONST.APPROVAL_WORKFLOW.TYPE.REMOVE ? '' : firstApprover.email ?? '';
```

Comment:
When removing the workflow, should we set the submitTo to default approver?

Reply:
We could, but couple of weeks ago we agreed with Tim Golen that it will be easier to use an empty string, [here](https://github.com/Expensify/App/issues/46984) is an issue where it was implemented on the BE.